internal/tiltfile/starkit/get_state.go

```diff
@@ -0,0 +1,22 @@
+package starkit
+
+import (
+	"fmt"
+	"reflect"
+
+	"go.starlark.net/starlark"
+)
+
+func GetState(t *starlark.Thread, v interface{}) (interface{}, error) {
```

Comment:
Ah, I only added this because I looked for an existing pattern and didn't find one. Even after looking, I only see one place that does this, and your reference to "all the existing code" makes me wonder if I'm not looking at the right thing? All the other calls I see are getting the state from the model from the result of ExecFile, not the thread.

Anyway, yeah, ModelFromThread -> GetState seems cleaner, and we can worry about StateFromThread later if we need it in more places.